daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/test_DFDL-714.txt

```diff
@@ -1 +1 @@
-Hello World 
+Hello World 
```

Comment:
I did not change this file. ONly moved it.

Reply:
Looking at the before and after files in a hex editor, it looks like a CRLF was replaced with an LF. Probably your IDE doing it, but it's a fine change. We can't rely on line endings anyways due to git auto-crlf, and I'd prefer things be LF anyways.